src/lib/draw/image/ImageMode.svelte

```diff
@@ -1,11 +1,11 @@
 <script lang="ts">
+  import { mode } from "lib/draw/stores";
```

Comment:
Should we use this or relative pathing everywhere? Is there a reason to use different ones in different places?

Reply:
Good point, using absolute everywhere (to make later file reorgs easier). Except when it's a "private" file to one directory, like draw/route/.